core/players-handbook/classes/class-ranger.xml

```diff
@@ -1017,4 +1022,25 @@
 			<description>When you cast a spell targeting yourself, you can also affect your beast companion with the spell if the beast is within 30 feet of you.</description>
 		</sheet>
 	</element>
+
+	<element name="Natural Explorer Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_NATURAL_EXPLORER">
+		<compendium display="false" />
+	</element>
+
+	<element name="Favored Enemy Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_FAVORED_ENEMY">
+		<compendium display="false" />
+	</element>
+
+	<element name="Primeval Awareness Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_PRIMEVAL_AWARENESS">
+		<compendium display="false" />
+	</element>
+
+	<element name="Hide in Plain Sight Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_HIDE_IN_PLAIN_SIGHT">
+		<compendium display="false" />
+	</element>
+
+	<element name="Ranger's Companion Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_BEAST_MASTER_RANGER_COMPANION">
+		<compendium display="false" />
+	</element>
```

Comment:
Internal elements dont show up in the compendium so we can save space by using the below
```suggestion
	<element name="Natural Explorer Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_NATURAL_EXPLORER" />
	<element name="Favored Enemy Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_FAVORED_ENEMY" />
	<element name="Primeval Awareness Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_PRIMEVAL_AWARENESS" />
	<element name="Hide in Plain Sight Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_HIDE_IN_PLAIN_SIGHT" />
	<element name="Ranger's Companion Feature Replacement" type="Grants" source="Internal" id="ID_INTERNAL_FEATURE_REPLACEMENT_RANGER_BEAST_MASTER_RANGER_COMPANION" />
```